content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -229,7 +229,7 @@ constructor, or just one method:
 [source,groovy]
 ----
 package org.foo
-class Utilities {
+class Utilities implements Serializable {
```

Comment:
I will try to add some verbiage around it, but I'm loathe to litter these documents with so many admonitions for every possible way to fuck up Groovy :slightly_frowning_face: :